views/halloffame.html

```diff
@@ -69,6 +83,81 @@
 			</section >
 			<section class="hof-stats right-side">
 				<h2>My Stats</h2>
+				<div class="stats-helper">
+					<p><span class="regNum"></span> regular HOF conversations</p>
+					<p><span class="gpNum"></span> gold plated conversations</p>
+					<p><span class="gNum"></span> golden conversations</p>
+				</div>
+				<div id="chart"></div>
+			    <script>
+				    var $window = $(window);
+		
+			      	(function(d3) {
+			        'use strict';
```

Comment:
Since d3 is included in the head, while this script exists in the body, this script should have access to the d3 as a global variable.  Therefore, it probably isn't necessary to have the wrapping function from line 95 to 159 `})(window.d3);`, unless you plan on using another variable named d3 elsewhere.  They could probably be removed.

Also, is there a particular reason why you included the `use strict` here?  I've seen it used in places where developers want to enforce certain behaviors and throw extra errors in special situations.  I don't think it's necessary for our project.